test/karma.conf.js

```diff
@@ -22,7 +21,7 @@ const browserStack = {
   // Since we have limited resources on BrowserStack we often time out on PRs.
   // However, BrowserStack rarely fails with a true-positive so we use it as a stop gap for release not merge.
   // But always enable it locally since people usually have to explicitly have to expose their BrowserStack access key anyway.
-  enabled: !CI || !isPR || process.env.BROWSERSTACK_FORCE === 'true',
+  enabled: true,
```

Comment:
I enabled Browserstack tests on PRs to verify if Safari doesn't fail. I'll revert this change after the PR is approved.